x-pack/modules/azure/configuration/logstash/azure.conf.erb

```diff
@@ -4,6 +4,7 @@
 
 input{
 <%=
+deprecation_logger.deprecated("The Azure module has been deprecated and may be removed in a future release.")
```

Comment:
Same as above:

 "The Azure module has been deprecated in favor of the Filebeat Azure module, and may be removed in a future release."